content/en/platform/corda/4.12/enterprise/node/operating/tvu/tvu-cli.md

```diff
@@ -0,0 +1,114 @@
+
+{{< note >}}
+`--error-dir-path` must be a directory or an exception is thrown.
+{{< /note >}}
+
+### -f, --config-file
+
+Absolute path to node's configuration file for reading configuration directly from a node. It requires `-b` or `--base-directory` option. If this parameter is not supplied, the default is `node.conf`.
+
+Example: `-b /corda/cordapp-template-java/build/nodes/PartyA -f /corda/cordapp-template-java/build/nodes/PartyA/some-other-node.conf`
+
+### -i, --id-load-file
+
+The utility can re-verify transactions supplied by the user. This parameter takes a file path to a text file containing transaction IDs or to a `.zip` file containing files with file names as transaction IDs. ID reverification does not happen if this option is absent.
+
+{{< note >}}
+`--id-load-file` must not be a directory or an exception is thrown.
+{{< /note >}}
+
+{{< note >}}
+Since the utility does not support transaction ID re-verification with progress loading and registration, the `-i` or `--id-load-file` parameter cannot be specified with `-l` or `--load-tx-time`.
+{{< /note >}}
+
+### -l, --load-file-path
+
+Takes a file path where the current progress will be stored and the last progress can be loaded from. If the given file is empty, then progress is not loaded from the file and the utility only writes the progress to it.
+
+If this parameter is not provided, then progress is logged on-screen and the utility starts processing transactions starting with the earliest transaction in the database as per transaction time.
+
+### --load-tx-time
+
+The utility can resume the transaction verification from transaction time provided using this parameter. The provided transaction time is compared against transaction recorded time. The utility loads transactions which have a transaction timestamp greater than or equal to the provided time.
+
+If this parameter is not provided, then the utility starts processing transactions from the earliest transaction in the database as per transaction time or loads progress from file if the `-f` option is specified.
+
+{{< note >}}
+`--load-tx-time` takes preference over the `-f` option in terms of progress loading.
+{{< /note >}}
+
+You can create a new time instant using any of the following:
+
+1. Provide seconds and nanos without spaces (remember, no spaces), for example, give information as:
+    * seconds=10
+    * seconds10
+    * seconds=10,nanos10
+    * seconds10,nanos=10
+    * seconds=10,nanos=10
+    * seconds10,nanos10
+    * seconds10nanos10
+2. Provide UTC timestamp in format: 2007-12-03T10:15:30.00Z. The string must represent a valid instant in UTC and is parsed using `DateTimeFormatter.ISO_INSTANT`.
+
+{{< note >}}
+When using H2 database, if the utility is not able to find the database from the provided `dataSource.url`, then H2's default behavior is to create a database at the `dataSource.url location`. To avoid this, append `;IFEXISTS=TRUE` to your `dataSource.url`.
```

Comment:
```suggestion
When using a H2 database, if the utility is not able to find the database from the provided `dataSource.url`, then H2's default behavior is to create a database at the `dataSource.url location`. To avoid this, append `;IFEXISTS=TRUE` to your `dataSource.url`.
```